### PR TITLE
flake8 misc/tests/*.py

### PR DESCRIPTION
**Pull request by taesupkim**
Related to issue #4185.

Fixed errors related to flake8

### CODE REVIEW

File: theano/misc/tests/test_gnumpy_utils.py [hunk 1]
```diff
@@ -44,9 +40,6 @@ def test2(shape=(3, 4, 5)):
     Make sure that the gnumpy conversion is exact from CudaNdarray to
     garray back to CudaNdarray.
     """
-    gpu = theano.sandbox.cuda.basic_ops.gpu_from_host
```
Comment (fvisin) [inline theano/misc/tests/test_gnumpy_utils.py]: Why did you remove these lines? Just don't assign the theano.function to `ii`.

Comment (taesupkim) [inline theano/misc/tests/test_gnumpy_utils.py]: The removed 3 lines, which are defining function `ii`, are not used in function `test2`.

Comment (fvisin) [inline theano/misc/tests/test_gnumpy_utils.py]: The test might be checking if the theano.function can be compiled. Just remove the final assignment please (i.e. call `theano.function` but don't assign it to `ii`)

File: theano/misc/tests/test_pycuda_theano_simple.py [hunk 2]
```diff
@@ -14,19 +14,19 @@
 import theano.sandbox.cuda as cuda_ndarray
 import theano.misc.pycuda_init
 
+import pycuda
+import pycuda.driver as drv
+import pycuda.gpuarray
+
 if not theano.misc.pycuda_init.pycuda_available:
```
Comment (fvisin) [inline theano/misc/tests/test_pycuda_theano_simple.py]: Travis fails if you import pycuda before checking `theano.misc.pycuda_init.pycuda_available` because it tries to execute the test without GPU.

Please revert the previous import order and add `# noqa` to the lines checking `theano.misc.pycuda_init.pycuda_available`. For example here should be:
```python
import theano.sandbox.cuda as cuda_ndarray
import theano.misc.pycuda_init

if not theano.misc.pycuda_init.pycuda_available:  # noqa
    from nose.plugins.skip import SkipTest
    raise SkipTest('Optional theano package cuda disabled')

import pycuda
import pycuda.driver as drv
import pycuda.gpuarray
```

Please do the same for the other files too. This should fix Travis error.

**Comment by taesupkim** [comment]
@fvisin Hope all done with this. Thanks!

**Comment by fvisin** [comment]
Thank you @taesupkim! 
Looks good to me and Travis passes. Merged.
